install.sh

```diff
@@ -18,8 +18,8 @@ if [ -z ${PIP} ]; then
     exit 1
 fi
 
-LATEST_TAG=$(curl -sSL https://api.github.com/repos/uclalibrary/fester/releases/latest | grep '"tag_name": ".\+"' | grep -o '[0-9]\+\.[0-9]\+\.[0-9]\+' ) && \
+LATEST_TAG=$(curl -sSL https://api.github.com/repos/uclalibrary/festerize/releases/latest | grep '"tag_name": ".\+"' | grep -o '[0-9]\+\.[0-9]\+\.[0-9]\+' ) && \
```

Comment:
Do you want to create an initial release in this repo so this will work?

Reply:
yeah, I was thinking that too. A simple tag would suffice, right?

Reply:
hmm, seems like it would have to be a release, actually

Reply:
Yes, I think it needs that tag and release. Maybe also write a little bit in the README documenting how a release should happen (since we won't be using the script from Fester any more)?